### ISSUE TITLE
game update not working

### ISSUE THREAD

**Issue by dusk77** [sudachi-emu/sudachi#188]
sudachi installs the update successfully 
but it doesnt actual run it 
it runs the base game

**Comment by 777joe** [comment]
Yes, I have experienced a similar problem on Android. When I apply the update, and play the game, the update works fine. But, after quitting Sudachi and opening the app again, the update is not applied, and the game is reverted to the base version, even though in the 'Add-on' section, the update is shown to be applied. Overwriting the update, works again, until I quit the app.

**Comment by cedibb** [comment]
Try putting your updates/dlcs in the same folder as your games.
It is a bug for sure, but doung that solved it for me.

**Comment by 777joe** [comment]
@cedibb Can confirm, this method works. Thanks
